.flowconfig

```diff
@@ -0,0 +1,13 @@
+[ignore]
+.*/node_modules/.*
+
+[include]
```

Comment:
Can you remove the unused headings like this one?

Reply:
Done!